tcms/core/tests/test_ajax.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+
+from tcms.core.ajax import strip_parameters
+
+
+class Test_strip_parameters(unittest.TestCase):
```

Comment:
Please do not use ``_`` in class name. In this case, for example, ``TestStripParameters`` is ok.